photo.go

```diff
@@ -1,21 +1,50 @@
 package main
 
 import (
+	"crypto/rsa"
 	"fmt"
 	"net/url"
 	"strings"
 	"time"
 
 	"github.com/aws/aws-sdk-go/aws"
 	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/cloudfront/sign"
 	"github.com/aws/aws-sdk-go/service/s3"
+
+	"github.com/globocom/gothumbor"
+	"log"
 )
 
-type ImgixPhoto struct {
+type RescaledPhoto struct {
 	Key     string
 	BaseUrl *url.URL
 }
 
+type ImgixRescaledPhoto struct {
+	*RescaledPhoto
+}
+
+type ThumborCommon struct {
```

Comment:
From what I see, this is only being used in one place `ThumborRaw`. Unless I'm mistaken, maybe we can just merge these into `ThumborRaw`. Unless you plan to add another Thumbor child class?

Reply:
yeah, i had originally used it in two places, but i realised that was no good as i finished implementing the ThumborCloudfront, will toss it since it's overzealous